### PR TITLE
Added staging build icon

### PR DESCRIPTION
**Pull request by jvisenti**

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -193,7 +193,7 @@ PODS:
   - React-jsinspector (0.61.5)
   - react-native-background-timer (2.2.0):
     - React
-  - react-native-config (1.2.1):
+  - react-native-config (0.11.7):
```
Comment (jvisenti) [inline ios/Podfile.lock]: I don't know why this is changing, but running `yarn install:pod` locally updates the lock file.

Comment (johnschoeman) [reply ios/Podfile.lock]: probably need to run `yarn install` first. A previous commit updated `react-native-config` to 1.2.1 and that probably what we want here.

**Review by johnschoeman — APPROVED** [review]
LGTM

**Review by JacobJaffe — APPROVED** [review]
🎉 terrific, thanks

File: ios/Podfile.lock (hunk 1 above)
Comment (jvisenti) [reply ios/Podfile.lock]: Ah that makes sense, all set now

**Pull request merged**